addons/pos_sale/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_crm_team_pos_manager,crm.team.pos.manager,model_crm_team,point_of_sale.group_pos_manager,1,1,1,1
+access_sale_sale_order_line,sale.order.line,sale.model_sale_order_line,point_of_sale.group_pos_user,1,0,0,0
```

Comment:
You try to introduce this here because during loading you saw an access error about it. However, it was in fact an issue in the loading of "order_count" of a model in the loyalty module.

Maybe it's best to just remove this access rights and put sudo in the calculation of order_count.